app/server/src/features/accounts/resolver.integration.test.ts

```diff
@@ -0,0 +1,139 @@
+import { toGlobalId } from '../utils';
+
+const server = getOrCreateServer();
+const testClient = createMercuriusTestClient(server);
+const prisma = getPrismaClient(server.log);
+
+const toUserId = toGlobalId('User');
+
+const userData = {
+    id: '4136cd0b-d90b-4af7-b485-5d1ded8db252',
+    email: 'sallySmith@test.com',
+    firstName: 'Sally',
+    lastName: 'Smith',
+    password: 'testPassword',
+    preferredLang: 'EN',
+};
+
+beforeAll(async () => {
+    // Attack relevant plugins
+    plugins.attachCookieTo(server);
+    plugins.attachMercuriusTo(server);
+
+    // hash password as they are never stored in plaintext on the DB
+    const encryptedPassword = await bcrypt.hash(userData.password, 10);
+
+    // Add user to test DB
+    await prisma.user.create({
+        data: {
+            ...userData,
+            password: encryptedPassword,
+        },
+    });
+});
+
+afterAll(async () => {
+    await prisma.user.deleteMany();
+    await prisma.$disconnect();
+    redisEmitter.close(() => {});
+    await server.close();
+});
+
+describe('account resolvers', () => {
+    describe('Query', () => {
+        describe('[me]', () => {
+            test('[me] query with no cookie should return null', async () => {
+                // Arrange
+                const query = 'query { me { id } }';
+
+                // Act
+                const queryResponse = await testClient.query(query);
+
+                // Assert
+                const expectedResponse = { data: { me: null } };
+                expect(queryResponse).toEqual(expectedResponse);
+            });
+            test('[me] query with non-existing account cookie should return null', async () => {
+                // Arrange
+                const query = 'query { me { id } }';
+                const token = await jwt.sign({ id: '' });
+                testClient.setCookies({ jwt: token });
+
+                // Act
+                const queryResponse = await testClient.query(query);
+
+                // Assert
+                const expectedResponse = { data: { me: null } };
+                expect(queryResponse).toEqual(expectedResponse);
+            });
+            test('[me] query with existing account cookie should return user id', async () => {
+                // Arrange
+                const query = 'query { me { id } }';
+                const user = toUserId(userData);
+                const token = await jwt.sign({ id: user.id });
+                testClient.setCookies({ jwt: token });
+
+                // Act
+                const queryResponse = await testClient.query(query);
+
+                // Assert
+                const expectedResponse = { data: { me: { id: user.id } } };
+                expect(queryResponse).toEqual(expectedResponse);
+            });
+        });
+    });
+    describe('User', () => {
+        describe('organizations', () => {
+            test('should return empty list of edges when user has no organizations', async () => {
```

Comment:
Maybe wrap this in describe "field (or model) resolvers" describe block?  Feels weird to go from one describe block named "Query" to one named "User."

It's also fine to leave as-is -- up to you.

Reply:
I wasn't sure the best way to go about it, I just tried following the structure of the resolvers themselves but it might make more sense to have it just under a sub describe block in query.